cmd/sqlflowserver/main_test.go

```diff
@@ -1724,10 +1725,6 @@ func TestEnd2EndMaxComputePAI(t *testing.T) {
 
 	go start(modelDir, caCrt, caKey, unitTestPort, false)
 	waitPortReady(fmt.Sprintf("localhost:%d", unitTestPort), 0)
-	err = prepareTestData(dbConnStr)
```

Comment:
So are we assuming the pre-existence of the test data?

Reply:
Yes. Recreating the dataset tables will clean the current permission settings, which cause other users may not have access to the dataset tables.